apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -13,7 +13,7 @@ type Properties = {
 	label: string;
 	onClick?: (() => void) | undefined;
 	type?: "button" | "submit";
-	variant?: "icon" | "secondary";
+	variant?: "icon";
```

Comment:
I prefer having variant primary, secondary, let's do that way and remove isPrimary